eventmesh-connectors/eventmesh-connector-http/src/main/java/org/apache/eventmesh/connector/http/sink/data/HttpConnectRecord.java

```diff
@@ -49,11 +50,16 @@ public class HttpConnectRecord {
      * @return the converted HttpConnectRecord
      */
     public static HttpConnectRecord convertConnectRecord(ConnectRecord record, String type) {
+        Map<String, ?> map = record.getPosition().getOffset().getOffset();
```

Comment:
A HashMap should never be named with `map`.

Reply:
Resolved at https://github.com/apache/eventmesh/pull/4837/commits/6f3b36162f964120099ffd63ff1bec9b96f87b77

Reply:
By the way, the logging can get the offset value too.

Reply:
How to get it?

Reply:
>By the way, the logging can get the offset value too.

I mean that Slf4j can output an offset value instead of an offsetMap. The way to get the offset value is the same as here.

The better way to get the offset value is to get("queueOffset") instead of using a iterator (since there may be more pairs in offsetMap in the furture).

Reply:
Please fix CI~

Reply:
> The better way to get the offset value is to get("queueOffset") instead of using an iterator (since there may be more pairs in offsetMap in the furture).

Although both `RocketMQSourceConnector` and `KafkaSourceConnector` use 'queueOffset' as the key, there is one exception: `S3SourceConnector` uses `position` as the key. If I directly retrieve the value using 'queueOffset', I may get a null value.

```java
// RocketMQSourceConnector and KafkaSourceConnector
 public static RecordOffset convertToRecordOffset(Long offset) {
        Map<String, String> offsetMap = new HashMap<>();
        offsetMap.put("queueOffset", offset + "");
        return new RecordOffset(offsetMap);
    }
// S3SourceConnector
private RecordOffset getRecordOffset() {
        Map<String, String> map = new HashMap<>();
        map.put(POSITION, String.valueOf(this.position));
        return new RecordOffset(map);
    }
```

Do you have any other suggestions?

Reply:
> Please fix CI~

These CI errors are very strange, and I'm not sure where to start. Could you help me take a look? 

1. License Check: Firstly, I executed the License Check in my own repository without any problems. Secondly, the error message shows `cloudevents-http-vertx-2.3.0` and `vertx-web-client-4.0.0`, both of which have been removed by me. I've updated the dependencies to `cloudevents-http-vertx-3.0.0` and `vertx-web-client:4.4.6`.
2. Build (xxxx, java): This CI error occurred during the execution of unit tests, but the error is in the `HttpSourceConnectorTest`!!! I have never modified its code. It might be some bizarre chemical reaction causing it to fail. The error message is as follows:
   ```shell
    > Task :eventmesh-connectors:eventmesh-connector-http:test

    HttpSourceConnectorTest > testPoll() FAILED
       org.apache.http.conn.HttpHostConnectException at HttpSourceConnectorTest.java:108
           Caused by: java.net.ConnectException at HttpSourceConnectorTest.java:108
   ```
    It seems that this error occurs only when I enable the webhook mode, because the webhook mode starts the HttpServer service. To quickly resolve this issue, I need to change the unit tests of the HttpSinkConnector to use the common mode. I'm unable to reproduce this scenario locally.

Reply:
> The license check will output unadded and unused artifacts.
When I used `./gradlew dependencies` to print the dependency tree, I couldn't find `cloudevents-http-vertx-2.3.0` and `vertx-web-client-4.0.0`.  I'll go ahead and add them back into known-dependencies and see if that fixes the problem.

Reply:
> You can edit other unit tests to fit your design.

I plan to directly adjust the `HttpSinkConnector` in the unit tests to use the common mode and comment out the code related to the webhook mode to eliminate CI errors. What do you think?

Reply:
OK

Reply:
You can add logs that throw stack trace to locate problems.

Reply:
OK, I created a new branch in my repository for testing